README.md

```diff
@@ -2,29 +2,25 @@
 
 Artsy's Design System
 
-## Meta
-
-- Storybook: [https://palette-storybook.artsy.net](https://palette-storybook.artsy.net)
-- Docs: [https://palette.artsy.net](https://palette.artsy.net)
+- Storybook: [https://palette-storybook.artsy.net](https://palette-storybook.artsy.net) the latest design system and components are available here 🙌🏻 we have a new repo for icons and prefer to use that one ⬇️ Icons will be removed from palette-storybook.artsy.net eventually
+- Icons: [https://icons.artsy.net/](https://icons.artsy.net/) use this for icons
+- Docs [https://palette.artsy.net](https://palette.artsy.net) To be depracated. It contains old design system and components. Docs will eventually move to the palette-storybook.artsy.net
```

Comment:
It doesn't contain "old" design system components, exactly; it's just a bit out of date and is rarely in sync.

Reply:
That's my definition of old in this case but I'm happy to update the phrasing 🙌🏻